common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -286,10 +287,38 @@ def _get_user(self):
         """
         Return the current runtime Django user.
         """
-        from django.contrib.auth.models import User
```

Comment:
not sure why this was being imported here?

Reply:
If it isn't erroring, I'd say you made an improvement!